wiki/russian/style-rus.css

```diff
@@ -0,0 +1,392 @@
+}
+.img3 {
+     border: 1px solid #55c5e9; /* Рамка вокруг фотографии */
+     background: #e8fbff; /* Цвет фона */
+     padding-left: 10px;
+     padding-bottom: 10px;
+     padding-right: 10px;
+     padding-top: 10px;
+     transition: 1s; /* Время эффекта */
+}
+.img3:hover {
+     transform: scale(1.2); /* Увеличиваем масштаб */
+}
+.p5 {
+     float: right; /* Обтекание справа */
+     width: 350px; /* Ширина колонки */
+     height: auto; 
+     margin-left: 20px;
+     margin-top: 10px;
+     padding-left: 10px;
+     border: 2px solid #bbbbbb; 
+     text-align: center;
+     background-color: rgb(248, 249, 250);
+}
+.img4 {
+     border: 1px solid #55c5e9; /* Рамка вокруг фотографии */
+     background: #e8fbff; /* Цвет фона */
+     margin-right: 5px; /* Отступ справа */
+     margin-bottom: 0px; /* Отступ снизу */
+     margin-left: 10px;
+     margin-top: 5px;
+     padding-left: 10px;
+     padding-top: 10px;
+     padding-right: 10px;
+     padding-bottom: 0px;
+     transition: 1s; /* Время эффекта */
+}
+.img4:hover {
+     transform: scale(1.2); /* Увеличиваем масштаб */
+}
+.h6 {
+     margin-right: 20px;
+     line-height: 0; 
+     padding-top: 20px;
+}
+.line7 {
+     border-bottom: 2px groove #bbbbbb; /* Линия между ссылками */
+}
+.h7 {
+     margin-right: 20px;
+     line-height: 0; 
+     padding-top: 20px;
+}
+.line8 {
+     border-bottom: 2px groove #bbbbbb; /* Линия между ссылками */
+}
+.p6 {
+     float: right; /* Обтекание справа */
+     width: 360px; /* Ширина колонки */
+     height: auto; 
+     margin-left: 10px;
+     margin-top: 10px;
+     padding-left: 0px;
+     border: 2px solid #bbbbbb; 
+     text-align: center;
+     background-color: rgb(248, 249, 250);
+}
+.img5 {
+     border: 1px solid #55c5e9; /* Рамка вокруг фотографии */
+     background: #e8fbff; /* Цвет фона */
+     margin-right: 10px; /* Отступ справа */
+     margin-bottom: 0px; /* Отступ снизу */
+     margin-left: 10px;
+     margin-top: 5px;
+     padding-left: 10px;
+     padding-top: 10px;
+     padding-right: 10px;
+     padding-bottom: 10px;
+     transition: 1s; /* Время эффекта */
+}
+.img5:hover {
+     transform: scale(1.2); /* Увеличиваем масштаб */
+}
+.h8 {
+     margin-right: 20px;
+     line-height: 0; 
+     padding-top: 20px;
+}
+.line9 {
+     border-bottom: 2px groove #bbbbbb; /* Линия между ссылками */
+}
+.h9 {
+     margin-right: 20px;
+     line-height: 0;
+     padding-top: 20px; 
+}
+.line10 {
+     border-bottom: 2px groove #bbbbbb; /* Линия между ссылками */
+}
+.p7 {
+     float: left; /* Обтекание справа */
+     width: 300px; /* Ширина колонки */
+     height: 360px;
+     border: 2px solid #bbbbbb; 
+     padding-top: 0px;
+     padding-right: 50px;
+     padding-left: 10px;
+     padding-bottom: 20px;
+     margin-top: 0px;
+     margin-bottom: 0px;
+     margin-right: 0px;
+     text-align: center;
+     border: none;
+     background-color: rgb(248, 249, 250);
+}
+.img6 {
```

Comment:
u need to think about class names better, coz img6-7-8-9, it doesnt give any chance to understand what is belongs to